node/core/approval-voting/src/lib.rs

```diff
@@ -2275,7 +2338,11 @@ async fn launch_approval<Context>(
 	validator_index: ValidatorIndex,
 	block_hash: Hash,
 	backing_group: GroupIndex,
+	span: &jaeger::Span,
```

Comment:
`background` task span has no parent . I would fix that and also possibly break it into 2 spans, one for getting availability data (PoV) and waiting for candidate validation to happen.  These operations depend on the network and parachain block size so it would be nice to see them.

Reply:
Sounds wise! Will look into this.

Reply:
Sorted.